tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,751 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    assert openai_api_key != "", "❌ Please set your OpenAI API key"
</pre><p>This line can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>